app/src/main/java/com/iic/shopingo/ui/request_flow/Views/CreateRequestListItemView.java

```diff
@@ -0,0 +1,103 @@
+package com.iic.shopingo.ui.request_flow.views;
+
+import android.app.Activity;
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.KeyEvent;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.WindowManager;
+import android.view.inputmethod.EditorInfo;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.LinearLayout;
+import butterknife.ButterKnife;
+import butterknife.InjectView;
+import butterknife.OnClick;
+import butterknife.OnEditorAction;
+import butterknife.OnFocusChange;
+import com.iic.shopingo.R;
+
+/**
+ * Created by assafgelber on 3/4/15.
+ */
+public class CreateRequestListItemView extends LinearLayout {
+  @InjectView(R.id.create_request_item_title)
+  EditText titleView;
+
+  @InjectView(R.id.create_request_remove_button)
+  Button removeButton;
+
+  public CreateRequestListItemView(Context context) {
+    super(context);
+    init();
+  }
+
+  public CreateRequestListItemView(Context context, AttributeSet attrs, int defStyleAttr) {
+    super(context, attrs, defStyleAttr);
+    init();
+  }
+
+  public CreateRequestListItemView(Context context, AttributeSet attrs) {
+    super(context, attrs);
+    init();
+  }
+
+  public static CreateRequestListItemView inflate(ViewGroup parent) {
+    CreateRequestListItemView itemView = (CreateRequestListItemView) LayoutInflater.from(parent.getContext())
+        .inflate(R.layout.create_request_list_item, parent, false);
+    return itemView;
+  }
+
+  private void init() {
+    LayoutInflater.from(getContext()).inflate(R.layout.create_request_list_item_children, this, true);
```

Comment:
I just went over the blog post you sent me and I see no reason for having two layouts, you can nest the EditText and Button in the `R.layout.create_request_list_item` layout.